### ISSUE TITLE
[Feature Suggestion] Allow midi-qol to automate the removal of effects (or stacks of effects)

### ISSUE THREAD

**Issue by tposney** [tposney/midi-qol#652]
In GitLab by @danz5000 on Jan 16, 2022, 19:19

Currently, midi-qol can very elegantly apply active effects onto characters.

But how cool would it be if the opposite was true, and if you could automate Lesser Restoration to remove a specific condition (as an example)?

This could work even better if it remove X stacks from a given effect, which would allow you to automate some pretty awesome homebrew effects from either monsters or players.

In short, I think this could be a really cool idea.

**Comment by tposney** [comment]
Currently this is handled by an on use macro. In general you'd need to present a dialog to choose which condition (out of many possibly) to remove.

I will have a look at what might be possible.
